src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.module.css

```
@@ -0,0 +1,159 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background: var(--color-gray-300);
+  grid-column: 1 / span 2;
+  grid-row: 1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+
+  .bleedBackground {
+    grid-column: 1 / span 3;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: auto 590px;
+  padding-top: 140px;
+  row-gap: 84px;
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding: 50px 0 80px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaTitle {
+  grid-column: 2 / span 4;
+  grid-row: 1;
+}
+
+.gridAreaParagraph1 {
+  grid-column: 6 / span 6;
+  grid-row: 1;
+}
+
+.gridAreaParagraph2 {
+  grid-column: 2 / span 5;
+  grid-row: 2;
+}
+
+.gridAreaImage {
+  grid-column: 8 / span 4;
+  grid-row: 2;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaImage {
+    text-align: right;
+  }
+}
+
+
+/* Component wrappers */
+
+.paragraph1Wrapper {
+  font: var(--font-body-large);
+  margin-top: 0;
+}
+
+.paragraph2Wrapper {
+  font: var(--font-body-small);
+  margin-top: 75px;
+}
+
+.paragraph2Wrapper :first-child {
+  margin: 0;
+}
+
+.paragraph2Wrapper *:not(:first-child) {
```

The `*` isn't needed on pseudo-classes.

```suggestion
.paragraph2Wrapper :not(:first-child) {
```